tests/NotificationEventSubscriberTest.php

```diff
@@ -36,12 +36,12 @@ public function toMail($notifiable): MailMessage
         return new MailMessage;
     }
 
-    public function onSent($channel): void
+    public function onSent(string $channel, $response): void
```

Comment:
`$response` must be nullable in here too.

Reply:
The $response variable is set to nullable.